dev-client/src/components/content/ScreenContentSection.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {ReactNode} from 'react';
+import {
+  Heading,
+  Box,
+} from 'terraso-mobile-client/components/NativeBaseAdapters';
+
+type Props = {
+  title?: string;
+  backgroundColor?: string;
+  children: ReactNode;
+};
+
+export const ScreenContentSection = ({
```

Comment:
very happy to see this :)

Reply:
Still don't know how to fully align with the hierarchy in Courtney's mental model. These are all I've got in mind, lmk if you have a preference or a better idea:

**Option 1**: (what exists currently)
- ScreenContentSection with title="Soil ID"
- ScreenContentSection for Soil Matches with grey background
- ScreenContentSection with title="Site Data"

**Option 2**: 
- One-off SoilIdSection component that exists only for conceptual reasons
    - ScreenContentSection with title="Soil ID"
    - ScreenContentSection for Soil Matches with grey background
- ScreenContentSection with title="Site Data"

Reply:
Ok chatted with Garo and we concluded: Just do Option 1 for now, but I created tech-debt item #1288 to track this

Reply:
Option 3 sounds ideal to me but it's fine with me to do that as a followup task.